.github/workflows/test_buttercup.yml

```diff
@@ -52,4 +52,4 @@ jobs:
 
     - name: Testing...
       run: |
-        make test-buttercup
+        npm run test-unsafe test/jest/buttercup.test.js
```

Comment:
The name should be `test-buttercup.test.js`? 🤔

Reply:
good spot! renamed it too